rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -279,6 +276,19 @@ class Publisher : public PublisherBase
     return this->publish(*msg);
   }
 
+  void
+  publish(const rcl_message_raw_t * raw_msg)
+  {
+    if (store_intra_process_message_) {
+      // not supported atm
+      throw std::runtime_error("storing raw messages in intra process is not supported yet.");
```

Comment:
Make sure to make an issue about this after merging, and consider putting a TODO in here.